docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,290 @@
+   "source": [
+    "#### Spin-dependent transmission values\n",
+    "\n",
+    "Now both $O^{cell}(\\lambda)=O^{0, cell}\\cdot\\lambda$ and $P^{^3He, cell}(t)$ are known. <br>\n",
+    "Following Refs (1, 3), the polarization-dependent transmission matrices of a polarized neutron beam through the ^3He cells can be computed by inserting \n",
+    "\n",
+    "$$\n",
+    "\\hat{T}^{\\pm, cell}(t,\\lambda) = T^g\\cdot e^{-O^{0, cell}\\cdot\\lambda\\pm O^{0, cell}\\cdot\\lambda\\cdot P^{^3He, cell}(t)}\n",
+    "$$\n",
+    "\n",
+    "into:\n",
+    "\n",
+    "$$\n",
+    "\\hat{T}^{p}=\n",
+    "\\begin{pmatrix} \n",
+    "T^{p,+}(t,\\lambda) & 0 & T^{p,-}(t,\\lambda) & 0 \\\\ 0 & T^{p,+}(t,\\lambda) & 0 & T^{p,-}(t,\\lambda) \\\\ T^{p,-}(t,\\lambda) & 0 & T^{p,+}(t,\\lambda) & 0 \\\\ 0 & T^{p,-}(t,\\lambda) & 0 & T^{p,+}(t,\\lambda)  \n",
+    "\\end{pmatrix}\n",
+    "$$\n",
+    "\n",
+    "\n",
+    "$$\n",
+    "\\hat{T}^{a}=\n",
+    "\\begin{pmatrix} \n",
+    "T^{a,+}(t,\\lambda) &  T^{a,-}(t,\\lambda) & 0 &0\\\\ T^{a,-}(t,\\lambda) &  T^{a,+}(t,\\lambda) & 0 & 0\\\\ 0&0& T^{a,+}(t,\\lambda) &  T^{a,-}(t,\\lambda) \\\\ 0&0& T^{a,-}(t,\\lambda) &  T^{a,+}(t,\\lambda)\n",
+    "\\end{pmatrix}\n",
+    "$$\n",
+    "\n",
+    "- Note: same function for both cells, but with different input parameters\n",
+    "- Note: in principle the incoming beam to the polarizer is unpolarized, so $\\hat{T}^{unpol-in}(\\lambda, t)$ isntead of $\\hat{T}^{\\pm, cell}(t,\\lambda)$ could be used. However, the results will be the same in this case without the need to differ between the cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a1aee42-2dab-438a-aef4-6d90f1818038",
+   "metadata": {},
+   "source": [
+    "#### Sample data correction\n",
+    "\n",
+    "For polarizing <sup>3</sup>He-cells without neutron flippers in the beamline, the correction matrix $\\hat{M}$  becomes $\\hat{M} = \\hat{T}^{a}\\hat{T}^{p}$. To correct the measured data I, we need to apply the inverse matrix to the data $(\\hat{M})^{-1}$:\n",
+    "\n",
+    "$$\n",
+    "\\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \\hat{T}^a(t,\\lambda)\\cdot\\hat{T}^p(t,\\lambda) \n",
+    "\\begin{pmatrix} S^{++}_{t,\\lambda} \\\\ S^{+-}_{t,\\lambda}  \\\\ S^{-+}_{t,\\lambda} \\\\  S^{--}_{t,\\lambda}\\end{pmatrix}\n",
+    "$$\n",
+    "\n",
+    "$$\n",
+    "\\left(\\hat{T}^a(t,\\lambda)\\cdot\\hat{T}^p(t,\\lambda)\\right)^{-1} \\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \n",
+    "\\begin{pmatrix} S^{++}_{t,\\lambda} \\\\ S^{+-}_{t,\\lambda}  \\\\ S^{-+}_{t,\\lambda} \\\\  S^{--}_{t,\\lambda}\\end{pmatrix}\n",
+    "$$\n",
+    "\n",
+    "- Note: probably easier to first put functions into the matrix product $T^a$ $\\cdot$ $T^p$, then numerically get the inverse of that matrix product.\n",
```

Comment:
If done *numerically* this will be wasteful, since the matrices have a very special structure and many zeros. Therefore, it should be done *analytically*.